src/MainWindow.vala

```diff
@@ -66,6 +66,7 @@ public class AppCenter.MainWindow : Gtk.ApplicationWindow {
                 this.maximize ();
                 break;
             default:
+                move (saved_state.window_x, saved_state.window_y);
```

Comment:
Add an if statement to check for defaults here so we don't move the window to 250x250 on the first launch - let the window manager handle that